### PR TITLE
[create] Fix bug with install deps

### PR DESCRIPTION
**Pull request by jorenbroekema**
fixes https://github.com/open-wc/open-wc/issues/284

It also allows the user to select the dependency manager.

Note: There is one problem where the prompt is fired twice, once before installing dependencies, and then at the end it prompts the user again. I am not sure why this is so perhaps one of you guys can look into that, I can't really figure it out.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,6 @@ Now that you have cloned the repository, run the following commands to set up th
 
 ```
 npm install
```
Comment (daKmoR) [inline CONTRIBUTING.md]:
```suggestion
yarn install
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,7 +20,6 @@ Now that you have cloned the repository, run the following commands to set up th
...
-npm run bootstrap
 ```
 
 Both commands will install packages like eslint which are necessary to make your changes.
```
Comment (daKmoR) [inline CONTRIBUTING.md]:
```suggestion
This will download and install all packages needed.
```

File: packages/create/src/core.js [hunk 3]
```diff
@@ -99,18 +96,13 @@ function _install(command = 'npm', options) {
   });
 }
 
-export async function installNpm(where) {
+export async function installNpm(where, packageManagerChoice) {
```
Comment (daKmoR) [inline packages/create/src/core.js]: can we also move the "decision" to the generator so this is
```js
export async function installNpm(where, command) {
```

I think that's easier to understand

Comment (jorenbroekema) [reply packages/create/src/core.js]: Not sure if I understood what you meant fully with move the "decision" but I changed it

Comment (daKmoR) [reply packages/create/src/core.js]: that was exactly what I had in mind 💪

**Comment by daKmoR** [comment]
wooohooo thank you for this contribution ❤️

**Pull request merged**